Gemfile

```diff
@@ -70,3 +70,4 @@ group :test do
   gem "selenium-webdriver"
   gem "webdrivers"
 end
+gem "rspec-rails", group: [:development, :test]
```

Comment:
Seems like you already have a development/test group, perhaps you could put this dependency in there to avoid duplication

Reply:
I moved it to a development/test group.